content/en/wastewater_dev.Rmd

```diff
@@ -394,8 +397,10 @@ Samples are screened daily for respiratory syncytial virus (RSV) during the wint
 The next plots illustrate the 7-day rolling mean in daily RSV wastewater viral signal. This number is the average of a week's daily readings. RSV is not a reportable disease, meaning that it does not fall under the list of diseases of public health significance in Ontario as designated by the Minister of Health$^1$. Nevertheless, RSV has been identified as a disease that could benefit from wastewater-based surveillance given its contribution to seasonal emergency department visits and hospitalizations among young children and to outbreaks in healthcare institutions such as long-term care homes. Visit [Respiratory Virus Detections](https://www.canada.ca/en/public-health/services/surveillance/respiratory-virus-detections-canada.html) in Canada for more information on RSV surveillance.
 
 ```{r, echo=FALSE, warning=FALSE, message=FALSE}
-reworked_figure(xaxis = "date", yaxis = list(daily_rsv_signal, rsv_7_day), titles = c(y = "Normalized RSV viral copies*", y2 = "", x = "Date", title = "<b>RSV wastewater viral signal, Ottawa</b>"), data = ott_covid_waste, date_constraint = TRUE, ticks = FALSE, constraint_val = 40, ticklabels = "%b %d")
-
+p <- reworked_figure(xaxis = "date", yaxis = list(daily_rsv_signal, rsv_7_day), titles = c(y = "Normalized RSV viral copies*", y2 = "", x = "Date", title = "<b>RSV wastewater viral signal, Ottawa</b>"), data = ott_covid_waste, date_constraint = TRUE, ticks = FALSE, constraint_val = 40, ticklabels = "%b %d")
+p <- layout(p, yaxis = list(range = c(0.00001275000, 0.0002740), exponentformat="none", tickformat=".5f"))
```

Comment:
The reasoning behind Liz saying she wanted the scale set to 0.00085 was purposely to zoom out - she's worried that when we zoom win and see the variation that hit invites more interpretation. We don't fully know what the signal means, and so it's more prudent to have the zoomed-out view for right now. 

So if we could set `range = c(0, 0.0008500)` that would be perfect, I think!